build.sbt

```diff
@@ -329,14 +330,14 @@ lazy val `kyo-bench` =
             libraryDependencies += "dev.zio"             %% "zio"                 % zioVersion,
             libraryDependencies += "dev.zio"             %% "zio-concurrent"      % zioVersion,
             libraryDependencies += "dev.zio"             %% "zio-prelude"         % "1.0.0-RC26",
-            libraryDependencies += "com.softwaremill.ox" %% "core"                % "0.0.25",
+            libraryDependencies += "com.softwaremill.ox" %% "core"                % "0.0.27",
```

Comment:
This is [breaking the build](https://github.com/getkyo/kyo/actions/runs/9255716594/job/25460096781?pr=426#step:4:52). We could make the ox upgrade separately.